README.md

```diff
@@ -22,7 +22,7 @@ technologies and produce Fedora CoreOS.
 
 # Get Fedora CoreOS
 
-Future Link to download page
+There is no official release yet, but there are [nightly images](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/) available.
```

Comment:
I think we should add some extra warning that these images are not stable, may not work, and can break without notice.

Reply:
Agreed.  Also, could we link to the parent directory instead?